### PR TITLE
{bio}[foss/2020b] isoCirc v1.0.4 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by zarybnicky**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by zarybnicky** [comment]
I'll re-do this tomorrow, so that is uses pyfaidx from the recipes and not as a bundle component.

File: easybuild/easyconfigs/i/isoCirc/isoCirc-1.0.4-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    ('pyfaidx', '0.6.2', {
+        'checksums': ['d1258f8d053cba0c90fe329254e8ec59eb28b535b48d9d06e8c7f1d74b8e4531'],
+    }),
+    ('pyinterval', '1.2.0', {
+        'modulename': 'interval',
+        'checksums': ['8c46224a05815affa803ed5620432236bba620a246701b860d98ae7358a41d21'],
+    }),
+    (name, version, {
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/Xinglab/isoCirc/archive'],
```
Comment (boegel) [inline easybuild/easyconfigs/i/isoCirc/isoCirc-1.0.4-foss-2020b.eb:52]: @zarybnicky Makes more sense to use the source tarball from PyPI? Cfr. https://pypi.org/project/isocirc

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/i/isoCirc/isoCirc-1.0.4-foss-2020b.eb (hunk 1 above)
Comment (zarybnicky) [reply easybuild/easyconfigs/i/isoCirc/isoCirc-1.0.4-foss-2020b.eb:52]: @boegel The PyPI archive does work, but it doesn't contain the example data that the GitHub does that are used in the sanity check. I'll keep using the GitHub archive unless that's what you want

Comment (boegel) [reply easybuild/easyconfigs/i/isoCirc/isoCirc-1.0.4-foss-2020b.eb:50]: OK, in that case it makes sense to use the source tarball from GitHub, thanks for clarifying.

File: easybuild/easyconfigs/i/isoCirc/isoCirc-1.0.4-foss-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    ('crlibm', '1.0.3', {
+        'checksums': ['48e17981f90d69c6bb0013f68bacbe7a157de864a533d15dd196ca7e98348a35'],
+    }),
+    ('gffutils', '0.10.1', {
+        'checksums': ['a8fc39006d7aa353147238160640e2210b168f7849cb99896be3fc9441e351cb'],
+    }),
+    ('mappy', '2.20', {
+        'checksums': ['00c8720783ac982b514b01b86349ede9809eeaccdf1328acb5a16dfeef51458b'],
+    }),
+    ('pyfaidx', '0.6.2', {
```
Comment (boegel) [inline easybuild/easyconfigs/i/isoCirc/isoCirc-1.0.4-foss-2020b.eb:41]: @zarybnicky Wasn't the idea to use the existing `pyfaidx` easyconfig for this? Or is a more recent version than 0.5.9.5 required?

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2695.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/b9917405d19e9dbffb84577776759052 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @zarybnicky!

**Pull request merged**